redfish-core/lib/redfish_sessions.hpp

```diff
@@ -37,10 +37,15 @@ inline void fillSessionObject(crow::Response& res,
     res.jsonValue["Name"] = "User Session";
```

Comment:
The line above needed bumped like it is upstream https://gerrit.openbmc.org/c/openbmc/bmcweb/+/56088/10/redfish-core/lib/redfish_sessions.hpp